glsk/glsk-quality-check-ucte/src/main/java/com/powsybl/glsk/ucte/quality_check/QualityReport.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.glsk.ucte.quality_check;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * @author Marc Erkol {@literal <marc.erkol at rte-france.com>}
+ */
+public class QualityReport {
```

Comment:
@geofjamg and @flo-dup I think that we have to use directly the functional logs design with a Reporter, as designed in the rest of the organization. What do you think?